src/components/Calculator.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Hi @karam084, Kindly fix Linters issues. We are not supposed to disable Linters. Linters are great tools that make sure that you are writing clean codes. They flag programming errors.

- I'll kindly advise you to extend React.PureComponent if you are struggling to resolve linters issues while using a class-base component, that's `class Calculator extends React.PureComponent {...`
- [ReadMore...](https://www.geeksforgeeks.org/reactjs-pure-components/)